README.md

```diff
@@ -3,7 +3,7 @@ Hi folks,
 My name is Roman Valihura. I'm the author of this extension. I'm Ukrainian.
 I was born in Ukraine. I'm living here at the moment.
 
-As you all know Russia invaded my country. 
```

Comment:
vscode format on save. Can revert if desired.

Reply:
Don't think is a problem since the line still there.